### DISCUSSION TITLE
Feature Request: Trigger on add

### DISCUSSION THREAD

**Discussion by jwfox5150** [meisnate12/Plex-Meta-Manager#1025] [Ideas]
I do not possess the coding knowledge to contribute to this, but I have thought of what I think is a decent idea. It would be so cool if we could tell PMM to monitor a library/libraries for new content, and then execute a portion of its script solely against the new content.

Example:
Dedicated 4K movie library has a new file added
PMM "watchdog" sees a movie has been added and triggers an overlay run on solely the new file

This would be nice because it would allow for immediate overlaying of new content's posters instead of relying on a cron job to take care of it.

**Comment by gitttt54** [comment]
Another way would be to run a cron-job every 30 mins with a config.yml which only handles the "recently added" movies in plex.
(If this is possible with PMM, I don't know...but those movies must have a special flag...)
